extensions/amp-carousel/0.2/test/test-e2e/test-arrows-in-lightbox.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {sleep} from './helpers';
+
+describes.endtoend(
+  'amp carousel in lightbox',
```

Comment:
```suggestion
  'amp carousel:0.2 in lightbox',
```

Reply:
WDYT about adding an equivalent test suite for amp-base-carousel?

Reply:
Is it recommended that devs use amp-base-carousel in a Lightbox?

Reply:
Yes both are possible.